ocfweb/docs/docs/staff/getinvolved.md

```diff
@@ -0,0 +1,227 @@
+﻿[[!meta title="Getting Involved"]]
+
+## How can I get involved in the OCF?
+
+Glad you asked! Students from all backgrounds are welcome to join staff. We
+don’t interview or have a selective process for joining-- you simply have to
+express interest and show up to our weekly meetings. If you want to learn more
+and be an active participant, there are multiple ways to get started:
+
+### Stay up-to-date
+
+We use a mailing list ([ocf.io/announce][announce]) to announce weekly meetings
+and recap past meetings. If you’re joining in the middle of the semester or
+missed a meeting, you can also read the mailing list archives to see what we
+went over in the past. This is a low volume mailing list (2 emails per week)
+and you can unsubscribe at any time.
+
+[[Read more about other mailing lists available to join|doc
+staff/mailing-lists]]
+
+### In-person meetings
+
+We hold two meetings every Monday:
+
+#### Board of Directors (BoD)
+
+The OCF Board of Directors has open meetings from 7-8 pm every Monday. BoD is
+where we discuss and vote on administrative decisions. If you are interested in
+gaining insight into the internal operations the OCF from an administrative
+perspective, please come to BoD meetings! Meetings always are open to all.
```

Comment:
I think that there should be more description on what happens at BoD meetings, to attract people potentially more into the administrative side of the OCF. I think there should be a bit longer elaborating on the third sentence, perhaps "If you are somebody who wants to figure out how an organization such as the OCF gets and spends its money, what issues the OCF needs to deal with, and all the administrative work that needs to be done to keep the OCF running, please come to BoD meetings!"

I think it would be good to make a list of administrative things that the OCF needs help with to prospective staff, much like the list of technical issues below. I don't believe myself qualified enough to make such a list, but perhaps such a list might include the following.

Finances -
Revenue - include a link to the Fall 2018 buysheet and link some spreadsheet summarizing how our money is being used, and possible ways we may want to spend money on (desktop refresh, more servers, HPC, more stuff to lend)
Income - Briefly summarize (or copy from some previous slides) our funding sources, talk about referendum, etc. Talk about how the OCF needs people to come up with new ways to spend money, and new ways to get money.

Collaborations -
List past collaborations and invite reader to help develop our relationships with various other campus groups and tech companies (Facebook, Cloudless, plus whatever collaborations we've done with campus groups). Insert something like "We need people to look into other collaborations."

Lab maintenance -
We need people to help write and decide on a banning policy, CLA, and figure out how to provide more services that the majority can use other than just printing (perhaps lending stuff?). We need people to talk with <insert long list of roles that keep the OCF running>.